account_invoice_constraint_chronology/i18n/ar.po

```diff
@@ -20,33 +20,62 @@ msgstr ""
 "&& n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5;\n"
 
 #. module: account_invoice_constraint_chronology
-#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_bank_statement_import_journal_creation__check_chronology
 #: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal__check_chronology
 msgid "Check Chronology"
 msgstr ""
 
 #. module: account_invoice_constraint_chronology
-#: code:addons/account_invoice_constraint_chronology/model/account_invoice.py:66
+#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
```

Comment:
do you know why the line number is wrong ?

Reply:
hum... good question probably related to my development environment...


Also I think the root matter here is that .pot file is still missing which is probably why it's not properly handle on weblate :thinking:

Reply:
ok

Reply:
infact .pot file looks good, the matter is probably about msgmerge not sure when it should be run between, I belives while merging PR before sending terms to weblate !?